darts/tests/models/forecasting/test_tsmixer.py

```diff
@@ -0,0 +1,601 @@
+from darts.logging import get_logger
+
+logger = get_logger(__name__)
+
+try:
+    import numpy as np
+    import pandas as pd
+    import pytest
+    import torch
+    from torch import nn
+
+    from darts import TimeSeries, concatenate
+    from darts.dataprocessing.transformers import Scaler
+    from darts.models.forecasting.tsmixer_model import TSMixerModel
+    from darts.tests.conftest import tfm_kwargs
+    from darts.utils import timeseries_generation as tg
+    from darts.utils.likelihood_models import GaussianLikelihood
+
+    TORCH_AVAILABLE = True
+
+except ImportError:
+    logger.warning("Torch not available. TSMixerModel tests will be skipped.")
+    TORCH_AVAILABLE = False
+
+
+if TORCH_AVAILABLE:
+
+    class TestTSMixerModel:
```

Comment:
```suggestion
    @pytest.mark.skipif(TORCH_AVAIBLE is False, reason="Torch not available. TSMixerModel tests will be skipped.")
    class TestTSMixerModel:
```

You can use this decorator and the code becomes more readable :-) 


and you can delete the ` if TORCH_AVAILABLE:` above